.gitignore

```diff
@@ -20,13 +21,15 @@ TAGS
 *.mp4
 
 # Docs
-docs/Manifest.toml
 docs/build/
 docs/src/generated/
 !docs/src/assets/*.png
 !docs/src/assets/*.svg
 !docs/src/images/*.png
 
+# Experiments
+!experiments/AMIP/*/Manifest.toml
```

Comment:
@LenkaNovak if there are experiments that require a manifest you can add a similar line to the .gitignore.